### PR TITLE
Moving test util method to test folder

### PR DESCRIPTION
**Pull request by asoto-iov**
PR related with RSKJ one: https://github.com/rsksmart/rskj/pull/1922

### CODE REVIEW

**Comment by asoto-iov** [comment]
rskj:remove_randomness_from_unit_tests

**Comment by Vovchyk** [comment]
pipeline:run

File: src/test/java/co/rsk/federate/signing/hsm/client/HSMSignatureTest.java [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 package co.rsk.federate.signing.hsm.client;
 
+import co.rsk.federate.signing.utils.TestUtils;
 import org.ethereum.crypto.ECKey;
 import org.ethereum.crypto.HashUtil;
```
Comment (marcos-iov) [inline src/test/java/co/rsk/federate/signing/hsm/client/HSMSignatureTest.java:23]: Is this still being used?

Comment (asoto-iov) [reply src/test/java/co/rsk/federate/signing/hsm/client/HSMSignatureTest.java]: No it isn't. I have disabled import optimizations in the IDE. Solved!

**Comment by asoto-iov** [comment]
pipeline:run

**Comment by asoto-iov** [comment]
pipeline:run

**Comment by asoto-iov** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by asoto-iov** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by asoto-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Review by marcos-iov — APPROVED** [review]

**Review by Vovchyk — APPROVED** [review]

**Pull request merged**